KalturaMediaServer/src/main/java/com/kaltura/media/server/managers/KalturaLiveManager.java

```diff
@@ -182,9 +177,29 @@ private void loadAssetParams() {
 			impersonateClient = null;
 		}
 
+		public synchronized void setEntryIsLive() {
+			if (!isReadyForPlayback()) {
+				logger.info("setEntryIsLive was called - IsLive is already on");
+				return ;
+			}
+			TimerTask setMediaServerTask = new TimerTask() {
+
+				@Override
+				public void run() {
+					logger.info("Initial timer task running [" + liveEntry.id + "] - PLAYABLE");
```

Comment:
so it looks like removing the an isRegistered() check here is needed after all since the task can start and the registered status can go to unregistered